### ISSUE TITLE
Galaxy Server upgrading failed

### ISSUE THREAD

**Issue by Dirowa** [Avans-ATGM/infrastructure#49]
Hi @hexylena , The galaxy upgrading of the server failed.
Currently, do i see in the logs a few errors which I was not able to solve.
It could also be because I messed it up with debugging. TLDR below

```
Sep 20 09:39:40 galaxy.atgm.avans.nl uwsgi[3382296]: --- no python application found, check your startup logs for errors ---
Sep 20 09:39:40 galaxy.atgm.avans.nl uwsgi[3382296]: [pid: 3382296|app: -1|req: -1/319] 145.48.205.20 () {38 vars in 444 bytes} [Tue Sep 20 09:39:40 2022] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
```

So far i did also create before upgrading the server create a backup of the database following documentation.
The database backup is stored in ``` //data/galaxy_backup/galaxy_database-2022-19.sql.gz ``` which is for galaxy 21.09 database version 179.

Further the database version is currently 180.

So here comes the part why I think I messed up.
Instead of going thorugh all the versions did i change the commit id from 21.09 directly too 22.05.
This resulted in an error where in jenkins was already an issue with an solution which i tried to perform manually.
After failing alot due dependencies (which i had to install manually with pip in the conda enviorment of galaxy) i failed and thought lets do first 22.01 version.

That one worked and the database was upgraded to version 180. Then i continued to version 22.05 where this error occours described above. When I try to downgrade the galaxy server back to 22.01, jobs will fail because of a mismatch in the galaxy database ( what kinda obvious is due there was a big change in the database type)

TLDR:
Galaxy 22.05 will not start up ``` Sep 19 17:17:20 galaxy.atgm.avans.nl uwsgi[1675977]: --- no python application found, check your startup logs for errors --- ```

Galaxy database backup version 179 exists in  ``` //data/galaxy_backup/galaxy_database-2022-19.sql.gz ``` 
Tried upgrading the galaxy database first manualy resulting in me had to install alot of dependencies by pip. did this in both conda enviorments as galaxy user.
after i upgraded the galaxy server to 22.01 which succeed then 22.05 by ansible. 

Galaxy server is possible broken because i installed dependencies manually and changed the alies of the galaxy user python path in hope of debugging.

**Comment by hexylena** [comment]
So, 22.05 also changed how deployment works which really complicates things for us, and we should hold off installing temporarily.

I would recommend that this week we try and downgrade again to 22.01

**Comment by Dirowa** [comment]
hi @hexylena  thank you for the quick response.
Is it okay if i plan a meeting with you in for Thurseday?

**Comment by hexylena** [comment]
do it

**Comment by Dirowa** [comment]
Galaxy server is fixed by using various tools
In total we had to plug an backup database (v.180) in the posgresql database.
Revert Galaxy to version 22.01
Ansible managed to upgrade the database to the correct version  of 180

What will be done
- Create a monitoring script to check if backups of the galactic database are made
- Create a tutorial when a disaster like this happends
